lib/uri.mli

```diff
@@ -0,0 +1,24 @@
+val get_sld : string -> string option
+(** Get the URI's second level domain, if it has one. *)
+
+val append_to_base : rel_path:string -> string -> string
+(** Append a relative path to a base URI. *)
+
+val chop_git_prefix : string -> string
+(** Chop the prefix "git+" from a URI, if there. *)
+
+val to_https : string -> string
+(** Convert "git\@" and "git+ssh://" into https URI's. Leave other URI's as are. *)
```

Comment:
I've looked this up and it seems like the "as is" expression is a fixed expression, so we should write either "as is" or "as they are" (slight preference for the shorter one)

Reply:
Instead of writing `"git\@"` (I guess it's an ocamlformat bug that rewrites `"git@"` into `"git\@"`), we should write `[git@]`, the `[]` notation preserves the input better

Reply:
Oh, that's interesting. I didn't know about the "as is" fixed expression. And also thanks for the ocamlformat tip!